packages/models/src/event.js

```diff
@@ -412,8 +412,12 @@ export default class Event {
 
   get qualifiedMethodId() {
     const { definedClass, isStatic, methodId } = this;
-    if (!definedClass) return undefined;
-    return `${definedClass}${isStatic ? '.' : '#'}${methodId}`;
+    let ret = `[${this.orig_id}] `;
```

Comment:
Does this only affect the UI or might it affect other code like event fingerprints as well?

Reply:
I don't really know. I suspect it could have an impact on other code.

I'm not really proposing these changes as-is. I mostly wanted to put them someplace others could use them to reproduce what I put in my doc.